src/jit/gentree.h

```diff
@@ -4932,7 +4930,7 @@ struct GenTreeStmt : public GenTree
         }
         else
         {
-            return gtNext->AsStmt();
+            return gtNext;
```

Comment:
The whole function is now just `return gtNext;`

Reply:
Yes, but I am planning to do it as a separate big renaming PR, where `GenTreeStatement` would be renamed to smth like `Statement`?